csharp-basics/exercises/Polymorphism/AdApp/NewspaperAd.cs

```diff
@@ -2,17 +2,18 @@ namespace AdApp
 {
     public class NewspaperAd : Advert
     {
-        private int column;
-        private int rate;
+        private int _column;
+        private int _rate;
 
-        public NewspaperAd(int fee) : base(fee)
+        public NewspaperAd(int fee, int column, int rate) : base(fee)
         {
+            this._column = column;
+            this._rate = rate;
         }
 
-        private new int Cost()
+        public override int Cost()
         {
-            var fee = base.Cost();
-            return fee;
+            return base.Cost() + _rate * _column;
```

Comment:
@EdgarsSalmins šis ir ok return base.Cost() ? Izskatās nepierasti

Reply:
man liekas ka ir fine... 
ja gribas uzlabot lasāmību tad var uztaisīt mainīgo ar baseFee = base.Cost() vai arī iznest metodē ārā => aka GetBaseFee;

Reply:
![image](https://user-images.githubusercontent.com/93127995/146594815-fd87e291-2a12-4f2c-8194-27550da019e4.png)
vai šādi der? iznesu mainīgo lasāmībai, vai savādāk?